modules/archived.js

```diff
@@ -2,9 +2,9 @@
 
 const LAST_GOOD_CRAWL_DATE = '20171011152352';
 
-function buildArchiveUrl(urlPath) {
+function buildArchiveUrl(urlPath, crawlDate = LAST_GOOD_CRAWL_DATE) {
```

Comment:
I think it's worth inlining this as `crawlDate = '20171011152352'`, one less bit of indirection.